script.js

```diff
@@ -0,0 +1,70 @@
+const add = document.querySelector('.button');
+const remove = document.querySelector('.close');
+const books = JSON.parse(localStorage.getItem('new-list')) || [
```

Comment:
- [ ] It would be better if the array of objects get displayed on the page once the page is loaded instead of adding the data before showing up all the array of objects